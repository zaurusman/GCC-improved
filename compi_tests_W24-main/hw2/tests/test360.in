qBpV	( true)			 			 

;byte  	x=aor (noti);
bool z
=false <= j+ 5b;if	(	(byte)  true 
 	or

		 
 (
 		 
( "you're"
  
  
/"a"  
 	 	
	))+ x(1b		  
  			 


 and	false))inty;if 	( "harry" /
	
	(	
  bool)"you're"	and a	
 
 	 	 
  	

 	

	  	 	 
	
	
	   
	    	
)while (
 	 "you're"and 3andtrue / 0  band"harry"- 	x( 


		"you're" )
 	 		
	 	*		
 	 	 

	

 	 
	 
	
		  
  
	
  
 
	
		i	(
		nottrue ) - 7 
	

	 			b +(	   int
 


  )4 *"a wizard" ortrue

and "you're"	 		 

 	 	
 	 
 	 
	* x- "awizard"ornot8b	
 	
 
 	
		
  
	 

) if (notfalse)bytelZLsO	  					
  	 		=3 	bor8bor aand "a	

   

 	  
	 		wizard";//Your_grammer_is_un_readable._:_(

 	
	 		
 ;
	 	 break;
byte x =0b   
 			
			
   
		 
	
	

 
		 

  
	
   
	


 	 ;{ 
			  				
while 

		( 2+not (	   int
)falseor z)intz=not	true 	
		 ;  
	 
				 
 		
 	  		
 
 

 	 		


	 
	 		
		}//Your_partner_looks_like_a_piazza_box._bitch.;	
	
		  
continue;		
	
 	   	
bool
  x;
  //Your_code_so_FAT_when_she_jumps_into_the_ocean_the_ocean_jumps_out._^^;break;			
			


	 

		
 //Your_github_hates_the_way_you_call_functions._^^;{


 break; 		 

 
}
y=j 
		

 	  
 	 		  		;//Your_laptop_is_none_existent._uwu		;if 	 (7b	
	  		 

	
 	 	 	 
			
 		
	 
		   
		
	 
or
not
(	( true 	

 
) *(
true)	or
5 

	
	   	  	 	 
	 	b 	and 5	b *false- "harry" ) 

 )z = true;	elsewhile ((	
  int  )"harry" )if	 


		  	  
 	  	  (i and	 		 	
	

	 	 	  	
	
  	 
 
		

 	 	 
 

	 	 

true) intzUjFG6yP78R= ( 0*2	 		
) ;elsex

	

 


  (a

);

 
 




  		
	 	   if	 
( 


     

 
 
	
 (i>= false and(notnotfalse 	  	 	and z* z ) ))	     int 

		

	 		
 
		 



i = false or 	 		 (			



					

 		    
 
	  	  		  				 
		
		 	
 int 

	
  )"awizard"/
 
		a(false-625b)
	
		
    *false/true>=true* 8continue	


	;
break;if ( 4)  

 	
		 
	




 	



   
		  
	  

	 "harry"continue


 	;elsetXnc=x ;{ while(3 elseb)	 			     		 	
	 break;break 	 ;
//Your_mama_so_FAT_when_she_jumps_into_the_ocean_the_ocean_jumps_out...;	 break3			
 
	
 	 
	
	 	;i
("awizard"	





 
	 	  	

	 )	 

    	;//Your_pp_looks_like_a_piazza_box._:_(; 	j	  	
   

	 (		
 
 	xand"c")
;}bytea 	;
 			
//Your_grammer_is_the_only_thing_more_ASHCENAZY_then_you._uwu;		if

 
 
  


(  	 "you're" 
	  		    )if(- true+ if
x-gL9Q51xCY(
 	y	
)
 
		
  { ==z
	 

 	

 )if( truex) z(	
	
	
			 	 "e"byte) ;elsebreak;continue
 


 	 
		
			
 
 
	  
 	
   ;
//Your_pp_is_smaller_then_required_to_pass_these_tests._:_(			 					 
	;continue;
continue;returnbreak;