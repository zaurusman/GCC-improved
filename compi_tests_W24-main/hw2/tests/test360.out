26: Exp -> TRUE
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
2: Statements -> Statement
17: Type -> BYTE
21: Exp -> ID
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
6: Statement -> Type ID ASSIGN Exp SC
3: Statements -> Statements Statement
18: Type -> BOOL
27: Exp -> FALSE
21: Exp -> ID
24: Exp -> NUM B
20: Exp -> Exp BINOP Exp
31: Exp -> Exp RELOP Exp
6: Statement -> Type ID ASSIGN Exp SC
3: Statements -> Statements Statement
17: Type -> BYTE
26: Exp -> TRUE
32: Exp -> LPAREN Type RPAREN Exp
25: Exp -> STRING
25: Exp -> STRING
20: Exp -> Exp BINOP Exp
19: Exp -> LPAREN Exp RPAREN
19: Exp -> LPAREN Exp RPAREN
24: Exp -> NUM B
27: Exp -> FALSE
29: Exp -> Exp AND Exp
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
20: Exp -> Exp BINOP Exp
30: Exp -> Exp OR Exp
line 20: syntax error
